speechbrain/utils/edit_distance.py

```diff
@@ -554,8 +561,12 @@ def wer_summary(details_by_utterance):
                 num_erraneous_sents += 1
         if dets["hyp_absent"]:
             num_absent_sents += 1
+    if num_scored_tokens != 0:
+        WER = 100.0 * num_edits / num_scored_tokens
+    else:
+        WER = 0.0
     wer_details = {
-        "WER": 100.0 * num_edits / num_scored_tokens,
```

Comment:
I think you should come back to the previous version to be more consistent with the SER...

Reply:
What previous version ? I don't get it

Reply:
You mean "WER": instead of WER = ?

Reply:
My bad forgot what I did there, so : I can't, your WER considers as a "well-placed word" an empty segment, instead of just ignoring it. It's a correction. Idk if your SER do the same error, never used it.